public/lang/en.json

```diff
@@ -487,14 +487,15 @@
   "A5E.MinimumD20Roll": "Minimum D20 Roll",
   "A5E.Movement": "Movement",
   "A5E.MovementSpeed": "Movement Speed",
-  "A5E.MovementHover": "Hover",
   "A5E.MovementConfigurationPrompt": "{name}: Configure Movement Speeds",
   "A5E.MovementBurrowingSpeed": "Burrowing Speed",
   "A5E.MovementBurrowingSpeedAbbr": "Burrowing",
   "A5E.MovementClimbingSpeed": "Climbing Speed",
   "A5E.MovementClimbingSpeedAbbr": "Climbing",
   "A5E.MovementFlyingSpeed": "Flying Speed",
   "A5E.MovementFlyingSpeedAbbr": "Flying",
+  "A5E.MovementHover": "Hover",
+  "A5E.MovementHoverAbbr": "(hover)",
```

Comment:
This is not an abbreviation, so we shouldn't be using the `Abbr` suffix for this key. I also don't think we need this key at all. The difference between this key and the one above is formatting.